account_receipt_print/views/report_receipt.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <data>
+        <template
+            id="report_receipt_document"
+            inherit_id="account.report_invoice_document"
+            primary="True"
+        >
+            <xpath expr="//h2" position="replace">
+                <h2>
+                <span
+                        t-if="o.move_type == 'out_receipt' and o.state == 'posted'"
+                    >Receipt</span>
+                <span
+                        t-if="o.move_type == 'out_receipt' and o.state == 'draft'"
+                    >Draft Receipt</span>
+                <span
+                        t-if="o.move_type == 'out_receipt' and o.state == 'cancel'"
+                    >Cancelled Receipt</span>
+                <span t-if="o.move_type == 'in_receipt'">Purchase Receipt</span>
+                <span t-if="o.name != '/'" t-field="o.name" />
+                </h2>
+            </xpath>
+
+            <div name="invoice_date" position="replace">
+                <div
+                    class="col-auto col-3 mw-100 mb-2"
+                    t-if="o.invoice_date"
+                    name="invoice_date"
+                >
+                    <strong>Receipt Date:</strong>
+                    <p class="m-0" t-field="o.invoice_date" />
+                </div>
+            </div>
+
+            <div name="due_date" position="attributes">
+                <attribute name="style">display: none;</attribute>
+            </div>
+
+            <p name="payment_communication" position="replace">
+                <p
+                    t-if="o.move_type == 'out_receipt' and o.payment_reference and o.payment_state == 'not_paid'"
+                    name="payment_communication"
+                >
+                Please use the following communication for your payment : <b><span
+                            t-field="o.payment_reference"
+                        /></b>
+                </p>
+            </p>
+
+            <p t-if="o.invoice_payment_term_id" position="attributes">
+                <attribute name="style">display: none;</attribute>
+            </p>
+
+            <xpath expr="//p[@t-if='o.narration']" position="before">
+                <t t-if="o.move_type in ('out_receipt', 'in_receipt')">
+                    <p class="mt32 mb16">
+                        <strong>Amount In Words:</strong>
```

Comment:
Cool add! But I'm wondering about whether it's not better keep it similar to `report_receipt_document` and move it to a customization by using i.e https://github.com/OCA/l10n-italy/pull/2841 Or is there some particular reason I'm not seeing?